caf-audit/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConstants.java

```diff
@@ -52,4 +52,11 @@
         public static final String DATE_SUFFIX = "_CADte";
     }
 
+    public final static class ConfigEnvVar {
+        public static final String ES_HOST_AND_PORT_VALS_ENV_VAR = "CAF_ELASTIC_HOST_AND_PORT_VALUES";
```

Comment:
Why have you used different variable names internally compared to the exposed externally as the Environment Variables?  This just adds to confusion and the internal names are neither notably shorter or longer and more descriptive than those external variable names, so I don't see any advantage, just confusion.